### ISSUE TITLE
Email server configuration in Docker

### ISSUE THREAD

**Issue by tetchegoyhen** [ELDAELRA/ELRI#56]
Email notifications do not work when using the Docker deployment.

**Comment by jcorteslinkare** [comment]
Hi Thierry,

As said in an earlier mail, i think it would be wiser for the partners to use an e-mail relay:

_I tried to configure a docker image with a SMTP server to receive/send e-mails but the best i could arrange (working one at least) was as an e-mail Relay (when not in relay mode we have unauthorized errors when sending to external e-mails (e.g. google accounts)).

As the configuration of a working e-mail docker image is going to be more complex than using an e-mail relay (e.g. google mail account, as it is free of charge), i think it is better for the partners to use an existing SMTP server for this purpose by defining it at the existing variables at the nrs_secret.properties file:

ELRI_EMAIL_TLS=True
ELRI_EMAIL_HOST=smtp.gmail.com
ELRI_EMAIL_PORT=587
ELRI_EMAIL_USER=jcortes@linkare.com
ELRI_EMAIL_PASS=*******

I tested it with a google account (needed to allow less secure apps) and it worked._
